packages/react/src/section-alert/docs/overview.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Section alert
+description: Section alerts are non-disruptive notifications that provide information and messages about a state change in a section of a page.
+group: Content
+storybookPath: /story/Content-SectionAlert--basic
+relatedComponents: ['page-alert']
```

Comment:
```suggestion
relatedComponents: ['page-alert', 'global-alert']
```

Reply:
Would you also be able to update page-alert and global-alert so they backlink to this component?